### PR TITLE
User invite flow is incorporated to the B2B sample application 

### PR DESCRIPTION
**Pull request by Achintha444**
Screenshot 2022-10-31 at 16 46 49

### CODE REVIEW

**Review by thivi — APPROVED** [review]

File: b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js [hunk 1]
```diff
@@ -97,10 +105,28 @@ export default function AddUserComponent(prop) {
...
     };
 
+    const inviteSelectOnChange = (value) => {
+        serInviteSelect(value);
+
+        switch (value) {
+            case InviteConst.INVITE:
+                setInviteShow(LOADING_DISPLAY_BLOCK);
+                setPasswordShow(LOADING_DISPLAY_NONE);
+
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js:119]: Shall we remove the unwanted new lines?

File: b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js [hunk 2]
```diff
@@ -276,3 +328,16 @@ export default function AddUserComponent(prop) {
...
+function EmailInvitePanel() {
+    return (
+        <Panel bordered>
+            <Stack spacing={ 30 }>
+                <EmailFillIcon style={ { fontSize: "3em" } } />
+                An email with a confirmation link will be sent to the provided
+                email address for the user to set their own password.
+            </Stack>
+
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js:340]: Is this new line needed? Please check other places also.

File: b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js [hunk 3]
```diff
@@ -20,18 +20,33 @@ import { setUsername } from "../../../util/util/apiUtil/decodeUser";
...
- * @returns `res` (if user added successfully) or `null` (if user addition was not completed)
- */
-export default async function decodeAddUser(session, firstName, familyName, email, username, password) {
-    const addUserEncode = {
+const InviteConst = {
+    INVITE: "pwd-method-invite",
+    PWD: "pwd-method-pwd"
+}
+
+
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js:28]: Let's remove these unwanted new lines

File: b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js [hunk 4]
```diff
@@ -45,7 +60,35 @@ export default async function decodeAddUser(session, firstName, familyName, emai
...
+        case InviteConst.PWD:
+            return pwdAddUserBody(firstName, familyName, email, username, password);
+
+        default:
+            break;
     }
+}
+
+/**
+ *
```
Comment (ThaminduR) [inline b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js:80]: Add a method description and add parameter descriptions. Check other applicable places as well.

File: b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js (hunk 1 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js:119]: It is an eslint rule to add a space before `break`

### Rule
- Expected blank line before this statement.eslint[@typescript-eslint/padding-line-between-statements](https://typescript-eslint.io/rules/padding-line-between-statements)

File: b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js (hunk 3 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js:28]: These have been fixed when this file was moved to a different location

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/user/controllerAddUser/controllerDecodeAddUser.ts

File: b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js (hunk 4 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js:80]: These have been fixed when this file was moved to a different location

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/user/controllerAddUser/controllerDecodeAddUser.ts

File: b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js (hunk 2 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js:340]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/ddf8187a9e62d909ef355a3b38a14e475a35c7aa

**Pull request merged**